AWGL/auto_qc
# bug fixes and formatting updates

to run unittests:

download repository and change into main folder.
run python manage.py test sample_sheet


to test index information changes:

load assay and referral type fixtures
run import_indexes management command using example file and command in top of script (upload tso500 and myeloid for comparison of index well or not)
view index sets using "indexes" link in footer and clicking through index set in table


to test family form:

after completing above steps, import a wings worksheet using the create a new worksheet and selecting wings shire query.
move to the worksheet clinical tab, select "add family" button to view form in popup. Select unique sample identifiers for each dropdown and click submit. family id should be rendered into information table
